osprey_ui/src/components/rules/RulesPage.module.css

```
@@ -0,0 +1,66 @@
+  min-width: 0;
+}
+
+.conditionBlock {
+  font-family: 'SFMono-Regular', Consolas, 'Liberation Mono', Menlo, monospace;
+  font-size: 12px;
+  padding: 8px 10px;
+  background: var(--background-secondary);
+  border: 1px solid var(--divider);
+  border-radius: 4px;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)